insitu/lib/insitu/filter.hpp

```diff
@@ -88,19 +87,27 @@ public Q_SLOTS:
 
 };    // class FilterWatchdog
 
+typedef enum FilterProps
+{
+    setToImageSize = 1,
+    keepAspectRatio = 2,
+    lockFilterProperties = 4
```

Comment:
Is there a reason for skipping?

Reply:
This is for bit flags, 1 = 0b0001, 2 = 0b0010, 4 = 0b0100, etc